client/packages/mobile/.gitignore

```diff
@@ -0,0 +1,17 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
+local.properties
+# Server android build, see package.json
+/server-lib
```

Comment:
mobile/package.json asks cargo to build things into this dir, easier to relocate to jniLibs this way